### PR TITLE
Symptom changes doublefixed

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
-nerfed sneezing
-small buff to hyphema
-fixed necrotizing fasciitis stealth scaling
## Why It's Good For The Game
-Hyphema is a fun symptom, but impossible to spread to its full potential without sneezing, even though it is possible to kill everyone quit easily without sneezing, oddly enough
-sneezing makes spreading a virus way too easy, especially for a starter symptom
-necrotizing fasciitis was supposed to scale inversely with stealth, according to the wiki, but does not. references to an unused Power value supported this
## Changelog
:cl:
balance: Nerfed Sneezing, now requires 9 transmission to spread
balance: Lowered hyphema symptom threshold to 11
tweak: necrotizing fasciitis inverse stealth scaling
:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Why not just add on to the previous pr

**Comment by zeskorion** [comment]
> Why not just add on to the previous pr

I have no idea how to do that

**Comment by qwertyquerty** [comment]
Just add more commits to your branch and they automatically show up

**Comment by qwertyquerty** [comment]
Thooloo if you merge this, use the squash and merge feature
image

File: code/datums/diseases/advance/symptoms/sneeze.dm [hunk 1]
```diff
@@ -27,20 +27,19 @@ Bonus
 	severity = 1
 	symptom_delay_min = 5
 	symptom_delay_max = 35
-	threshold_desc = "<b>Transmission 9:</b> Increases sneezing range, spreading the virus over a larger area.<br>\
+	threshold_desc = "<b>Transmission 9:</b> The host will sneeze periodically, spreading the disease. <br>\
 					  <b>Stealth 4:</b> The symptom remains hidden until active."
 
 /datum/symptom/sneeze/Start(datum/disease/advance/A)
-	if(!..())
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: don't remove this or the `return`

**Comment by zeskorion** [comment]
merge me nerd

**Comment by qwertyquerty** [comment]
1. You added like 4 random commits
2. You can remove the bee edit comments now that we have demodularized

**Review by CthulhuOnIce — DISMISSED** [review]
Remove bee edit comments

**Comment by zeskorion** [comment]
Removed the bee edit flags

**Comment by qwertyquerty** [comment]
Can you remove those 4 random commits you added

**Pull request closed without merging**